### ISSUE TITLE
Next.js buld. TradingView is not defined for SymbolOverview component

### ISSUE THREAD

**Issue by dzianislysionak** [JorrinKievit/react-ts-tradingview-widgets#15]
I am using Next.js. Everything works fine for dev mode. 
The application crush for "SymbolOverview" component if "npm run build" --> npm start -->ReferenceError: TradingView is not defined
Steps to reproduce:
- npx create-next-app@latest --ts
- npm i react-ts-tradingview-widgets
 pages/index.tsx :
`import type { NextPage } from "next";
import { SymbolOverview } from "react-ts-tradingview-widgets";

import styles from "../styles/Home.module.css";

const Home: NextPage = () => {
  return (
    <div className={styles.container}>
      <SymbolOverview symbols={[["BIB"]]} width="100%" />
    </div>
  );
};

export default Home;`

- npm run build
- npm start
![image](https://user-images.githubusercontent.com/59472058/194218732-9486c1d5-451d-4e22-bc4b-edfb3646a518.png)

**Comment by JorrinKievit** [comment]
Hey, will look into this when I get home from work.

**Comment by JorrinKievit** [comment]
Not sure how to fix this yet, but for now you can manually load the needed script in the _document.tsx file:

`import { Html, Head, Main, NextScript } from "next/document";
import Script from "next/script";

export default function Document() {
  return (
    <Html>
      <Head>
        <Script
          src="https://s3.tradingview.com/tv.js"
          strategy="beforeInteractive"
        />
      </Head>
      <body>
        <Main />
        <NextScript />
      </body>
    </Html>
  );
}`

**Comment by JorrinKievit** [comment]
What is the difference with your code and this? -> https://github.com/JorrinKievit/react-ts-tradingview-widgets/blob/main/src/components/Widget.tsx with https://github.com/JorrinKievit/react-ts-tradingview-widgets/blob/main/src/components/SymbolOverview.tsx

The scripts required by the widgets are not loaded by NextJS because of server side rendering

**Comment by JorrinKievit** [comment]
After consideration I think you should disable SSR and render the charts client side, example below:

```
const Home: NextPage = () => {
  const SymbolOverviewWidget = dynamic(
    () =>
      import("react-ts-tradingview-widgets").then((mod) => mod.SymbolOverview),
    { ssr: false }
  );

  return (
    <div className={styles.container}>
      <SymbolOverviewWidget colorTheme="dark" />
    </div>
  );
};

export default Home;
```

**Comment by JorrinKievit** [comment]
I don't know how I looked over this for so long, fixed in version 1.0.12. You should still import the charts dynamically -> 

```
const SymbolOverviewWidget = dynamic(
  () => import("react-ts-tradingview-widgets").then((w) => w.SymbolOverview),
  {
    ssr: false,
  }
);```

**Comment by dzianislysionak** [comment]
Thanks a lot. For me iin v 1.0.12 it working even without dynamic import
